### DISCUSSION TITLE
How to add a keybind to toggle a given bar's visibility?

### DISCUSSION THREAD

**Discussion by lominoss-git** [qtile/qtile#4247] [Q&A]
I am just looking for a way to toggle bar visibility, I read the documentation but I couldn't find a lazy function that does what I want.

**Comment by ShashwatAgrawal20** [comment]
> I am just looking for a way to toggle bar visibility, I read the documentation but I couldn't find a lazy function that does what I want.

```python
Key([mod], "f", lazy.hide_show_bar("top"), desc="Toggle fullscreen"),
```
Where top shows the position of your bar

**Comment by ShashwatAgrawal20** [comment]
Also, I wonder, If there is a way to **toggle the bar** for a particular workspace only.

**Reply by elParaguayo** [reply]
Use a hook which is called whenever you change group.

**Reply by ShashwatAgrawal20** [reply]
I want the bar to be hidden only in the workspace I have toggle on **full screen mode**. 

I have tried many things but it didn't worked.
